Automattic/wp-calypso
# Limit plugin title length on some elements

Fixes [67890](https://github.com/Automattic/wp-calypso/issues/67890)

## Proposed Changes

* in breadcrumbs, the name is truncated after 50 chars
* on the custom domain dialog, the title is now styled bold
* the custom domain dialog CTA button text is changed to "Install"

## Testing Instructions



* Go to a plugin page with a long name, eg. `/plugins/all-in-one-seo-pack/:site`
* Check that the plugin name in the breadcrumbs in truncated
* Click on Install
* Check the plugin title on the custom domain modal
* Check that the CTA button on the modal has the text "Install" instead of "Install {plugin name}"

| Before | After |
| ------ | ----- |
| ![CleanShot 2023-03-08 at 17 52 30@2x](https://user-images.githubusercontent.com/11555574/223791489-8d0605ca-030a-4e23-8e3c-5c1806102f75.png) | ![CleanShot 2023-03-08 at 17 52 08@2x](https://user-images.githubusercontent.com/11555574/223791500-f3fa4522-47ab-44ad-b547-a86cdbe06bd3.png) | 

## Pre-merge Checklist
